### PR TITLE
Car object

### PR DESCRIPTION
**Pull request by raghav-codes**
Added get() & post() endpoints, added IT & service test along with restDocs

### CODE REVIEW

**Review by maanick — APPROVED** [review]
Approved

File: src/main/java/com/example/speedwayapi/car/RaceCarController.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        return "{}";
+    }*/
+
+    @GetMapping("race-cars")
+    public List<RaceCarDto> getAllRaceCars() {
+        return raceCarService.getAllRaceCars();
+    }
+
+
+    @PostMapping("add-race-car")
```
Comment (davidgrieser) [inline src/main/java/com/example/speedwayapi/car/RaceCarController.java:27]: Does this conform to RESTful HTTP Verbs?

File: src/test/java/com/example/speedwayapi/car/RaceCarControllerIT.java [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+                .andExpect(status().isCreated());
+
+
+
+        mockMvc.perform(
+                post("/add-race-car")
+                        .content(objectMapper.writeValueAsString(input1))
+                        .contentType(MediaType.APPLICATION_JSON)
+        )
+                .andExpect(status().isConflict());
```
Comment (davidgrieser) [inline src/test/java/com/example/speedwayapi/car/RaceCarControllerIT.java:112]: REST Docs for conflict response?

**Review by davidgrieser — COMMENTED** [review]
Just some thoughts about HTTP and Docs

**Review by VaradharajanThiagarajan — APPROVED** [review]
LGTM

**Pull request merged**